states/idle/gamen.mod.py

```diff
@@ -19,7 +18,7 @@ def get_response():
         except requests.RequestException:
             return {}
         if not response.ok:
-            return response["error"]
+            print("Oei: " + response["error"])
```

Comment:
Dit zou dan response.json()["error"] moeten zijn. Iets printen in een module is niet handig want dat wordt naar ledcat gestuurd waardoor de display in de war raakt.

Reply:
Oei, in dat geval is ie in de meest recente commit weggehaald.